lib/ex_unit/lib/ex_unit/callbacks.ex

```diff
@@ -3,29 +3,43 @@ defmodule ExUnit.Callbacks do
   Defines ExUnit callbacks.
 
   This module defines both `setup_all` and `setup` callbacks, as well as
-  the `on_exit/2` function.
+  the `on_exit/2`, `start_supervised/2` and `stop_supervised/1` functions.
 
   The setup callbacks are defined via macros and each one can optionally
   receive a map with metadata, usually referred to as `context`. The
   callback may optionally put extra data into the `context` to be used in
   the tests.
 
-  The `setup_all` callbacks are invoked only once to setup the test case before any
-  test is run and all `setup` callbacks are run before each test. No callback
+  The `setup_all` callbacks are invoked only once per module, before any
+  test runs. All `setup` callbacks are run before each test. No callback
   runs if the test case has no tests or all tests have been filtered out.
 
+  `start_supervised/2` is used to start processes under a supervisor. The
+  supervisor is linked to the current test and the supervisor as well as
```

Comment:
I am not even sure what I wanted to say now. 😅 I will review accordingly, thanks!